code/__DEFINES/_subsystems.dm

```diff
@@ -122,6 +122,7 @@
 #define FIRE_PRIORITY_AUTOFIRE 450
 #define FIRE_PRIORITY_OVERLAYS 500
 #define FIRE_PRIORITY_EXPLOSIONS 666
+#define FIRE_PRIORITY_SILO 667
 #define FIRE_PRIORITY_TIMER 700
```

Comment:
ok but not higher than explosions wtf
make it like points tier-ish

Reply:
I've never touched this before. What are the consequences of having it at this level? My logic is its very important gameplay wise, but uses up very little processing time (it doesn't do much and only fires once a minute).

Reply:
It determines how "important" it is when master cant run everything
IE here it decides if it only has enough time for 1 it will run silos instead of explosions which will wait until next tick